backend/tests/integration/records/test_utils.py

```diff
@@ -94,7 +94,8 @@ def test_get_parent_record_when_more_than_one(inspire_app):
     }
     rec = create_record("lit", data=data)
     extracted_parent_record = get_parent_record(rec)
-    assert extracted_parent_record == parent_record
+    parent_records = list(parent_record, second_parent_record)
```

Comment:
do I miss something or this should throw an exception? :/

Reply:
Why do you think that?

Reply:
```
----> 1 list('a', 'b')

TypeError: list() takes at most 1 argument (2 given)
```

Reply:
Ah, yest... Sorry... Monday morning....